### ISSUE TITLE
Conduct 3-5 Usability Tests

### ISSUE THREAD

**Issue by noahmanger** [18F/openFEC#472]
**Goal:** Using the usability script, conduct 3-5 tests with desired users to test the a) home page, b) search, c) browsing/filters, d) glossary, e) single entity view.

**Tasks:**
- [x] Add glossary task to script
- [x] Email interview subjects to find testers
- [x] Test 1
- [x] Test 2
- [x] Test 3
- [x] Test 4
- [ ] Test 5
- [x] Process notes

**Comment by noahmanger** [comment]
Ok, I completed 4 and think that's enough for this sprint. There's another one coming on Friday, but would be nice to have some of the changes in place for that. 

I've created a new Trello board for our team for tracking usability findings: https://trello.com/b/NlIRzjFg/usability-findings

I'm using labels to signify the number of times something has come up, and moving those that are repeated to the top of the list.

![screen shot 2015-03-23 at 4 20 45 pm](https://cloud.githubusercontent.com/assets/1696495/6792744/938e95cc-d178-11e4-99ee-8b62f1b16edb.png)
